cmd/ftl/cmd_schema_get.go

```diff
@@ -26,25 +27,41 @@ func (g *getSchemaCmd) Run(ctx context.Context, client ftlv1connect.ControllerSe
 	if g.Protobuf {
 		return g.generateProto(resp)
 	}
+	remainingNames := make(map[string]bool)
+	for _, name := range g.Modules {
+		remainingNames[name] = true
+	}
 	for resp.Receive() {
 		msg := resp.Msg()
 		module, err := schema.ModuleFromProto(msg.Schema)
-		if err != nil {
-			return fmt.Errorf("%s: %w", "invalid module schema", err)
+		if len(g.Modules) == 0 || remainingNames[msg.Schema.Name] {
+			if err != nil {
+				return fmt.Errorf("%s: %w", "invalid module schema", err)
+			}
+			fmt.Println(module)
+			delete(remainingNames, msg.Schema.Name)
 		}
-		fmt.Println(module)
 		if !msg.More {
 			break
 		}
 	}
+	for name := range remainingNames {
+		fmt.Fprintf(os.Stderr, "module %s not found\n", name)
```

Comment:
This is not formatted consistently with FTL's CLI, but just return it as an error listing all missing names, and you'll get that for free.

Reply:
Oh right it should go to stderr. `--protobuf` now has the same behaviour